File: api/Bussiness/Services/LoginService/Dtos/LoginDto.cs
using Bussiness.Services.RoleService.Dtos;
using Bussiness.Services.UserService.Dtos;

namespace Bussiness.Services.LoginService.Dtos
{
    public class LoginDto
    {
        public UserDto UserDto { get; set; }
        public RoleDto RoleDto { get; set; }
        public bool? IsLoggedIn { get; set; }
        public bool RememberMe { get; set; }
    }
}

File: ui/Components/Login/Dtos/LoginDto.ts
import { initialRoleDto, RoleDto } from "../../Admin/AddRole/Dtos/roleDto";
import { initialUserDto, UserDto } from "../../Admin/AddUser/Dtos/userDto";

export interface LoginDto {
    userDto:UserDto 
    roleDto: RoleDto;
    isLoggedIn?: boolean ;
    rememberMe?:boolean 
}
export const initialLoginDto:LoginDto={
    userDto:initialUserDto,
    roleDto: initialRoleDto,
    isLoggedIn: false,
    rememberMe: false,
}
